src/packages/cli/src/__tests__/__snapshots__/init.test.ts.snap

```diff
@@ -8,9 +8,9 @@ Environment variables loaded from .env
 
 Next steps:
 1. Set the DATABASE_URL in the .env file to point to your existing database. If your database has no tables yet, read https://pris.ly/d/getting-started
-2. Set the provider of the datasource block in schema.prisma to match your database: postgresql, mysql, sqlserver or sqlite.
-3. Run prisma db pull to turn your database schema into a Prisma data model.
-4. Run prisma generate to install Prisma Client. You can then start querying your database.
+2. Set the provider of the datasource block in schema.prisma to match your database: postgresql, mysql, sqlite, sqlserver (Preview) or mongodb (Preview).
+3. Run prisma db pull to turn your database schema into a Prisma schema.
```

Comment:
Technically "data model" is correct - it is the part of the schema file with `model`s inside.

But probably fine.